cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/NoOpBrokerSetAssignmentPolicy.java

```diff
@@ -15,26 +16,25 @@
 
 
 /**
- * For any non-mapped Brokers, this policy does not assign any broker set.
+ * For any non-mapped Brokers, this policy assign it to an "unmapped" broker set.
  * Works as a strict broker set mapping policy where only the assignments that are pre-defined are the true source of truth.
  */
 public class NoOpBrokerSetAssignmentPolicy implements BrokerSetAssignmentPolicy {
-  private static final String UNMAPPED_BROKER_SET_ID = "unmapped";
+  public static final String UNMAPPED_BROKER_SET_ID = "unmapped";
 
   /**
    * Does not assign any broker set to non-mapped brokers.
    *
    * @return A map of broker Ids by their broker set Id
    */
   @Override
```

Comment:
should this be @Deprecated ?

Reply:
Now it is still the default BrokerSetAssignmentPolicy. Why do you think we should deprecate it?

Reply:
assignBrokerSetsForUnresolvedBrokers method is using ClusterModel argument here. I assumed you want the new rackIdToBrokerId mapping to be used as an argument going further. Ignore if the ClusterModel argument is still going to be used.

Reply:
Ah I see your point. The method was already deprecated at the interface level. I think it should be good enough?